src/main/java/mate/jdbc/service/DriverService.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverService {
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);
```

Comment:
```suggestion
    Driver get(Long id);
```
get() methods in Dao layer should return Optional (not in services). In the service layer better call method get() on Optional and return the object. It was written in checklist. Even in readme file method looks like:
Driver get(Long id);